install/kubernetes/cilium/charts/operator/templates/deployment.yaml

```diff
@@ -195,7 +171,7 @@ spec:
 {{- if .Values.global.etcd.managed }}
       # In managed etcd mode, Cilium must be able to resolve the DNS name of
       # the etcd service
-      dnsPolicy: ClusterFirstWithHostNet
+      dnsPolicy: {{ .Values.global.operator.dnsPolicy | default "ClusterFirstWithHostNet" }}
```

Comment:
In this commit there are a lot of changes that should not have been made.

Reply:
I change 1 line. What other changes?